.release-please-manifest.json

```diff
@@ -1,7 +1,7 @@
 {
-  ".": "5.6.0",
-  "packages/cli": "5.6.0",
+  ".": "5.7.0",
+  "packages/cli": "5.7.0",
   "packages/core": "5.6.0",
```

Comment:
interesting that it didnt bump this

Reply:
huh - it even released during your practice releases.. that is very strange.